common-services.yml

```diff
@@ -21,9 +22,10 @@ services:
 
   base-cache:
     image: redis:5.0.5
+    restart: always
```

Comment:
lets keep this in content node only. this is due to a content node limitation. if redis restarts the server also needs to restart